src/lib/utils/tokens.ts

```diff
@@ -161,6 +161,7 @@ export function isAvailableForSwap(token: Token): boolean {
 
 export function isIndexToken(token: Token): boolean {
   if (token.symbol === IndexToken.symbol) return false
+  if (token.symbol === HighYieldETHIndex.symbol) return true
```

Comment:
Without this hyETH would not be recognized as index because it's currently not listed in the indices token list. Probably in the future to be solved differently, if we would have lots of presale tokens (maybe an extra list?).